dm/dm.go

```diff
@@ -142,6 +161,10 @@ func execute() {
 
 	switch args[0] {
 	case "templates":
+		if len(args) < 2 {
+			fmt.Fprintln(os.Stderr, "No registry name supplied")
```

Comment:
We should be able to use the value of *template_registry, instead of args[1]. It defaults, so if the user doesn't specify it, we're still good.